.circleci/config.yml

```diff
@@ -92,3 +92,4 @@ workflows:
             branches:
               only: 
                 - /^.*main.*/                                
+                - MOB-1759_Implement_Bookmark_Export_Import
```

Comment:
I guess this should be removed before merging, but probably still waiting on some feedback from Kunle before that

Reply:
Yes right 👍  I would wait until the QA is done with this. Thanks again for pointing this out, let's keep this comment here for visibility. I'll also add this to the PR description.